packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -98,15 +98,29 @@ export function TelemetryView(): React.ReactElement {
 
 	return (
 		<Stack>
-			<StackItem>
-				<ListLengthSelection
-					currentLimit={maxEventsToDisplay}
-					onChangeSelection={(key): void => setMaxEventsToDisplay(key)}
-				/>
-			</StackItem>
 			<StackItem>
 				{telemetryEvents !== undefined ? (
-					<FilteredTelemetryView telemetryEvents={telemetryEvents} />
+					<>
+						<table>
```

Comment:
I wouldn't necessarily block the rest of the PR on this, but it's basically using a table for layout/structure, which we should avoid if possible. A pair of divs with the correct styles (`display: inline` or floating them probably part of it) should do the trick.